discord/api/dataConverters.py

```diff
@@ -1,18 +1,10 @@
 import inspect
 import typing
 
-from ..activity.activity import Activity
```

Comment:
where are all these imports removed (I assume you used optimize imports)?

Reply:
> I assume you used optimize imports

Yeah